'send_only' is deprecated and is now the default. In CI, use 'wait_for_results none' instead
Saving last configuration file to .certora_internal/25_09_21_11_29_32_207/run.conf
There is no TAC file. Going to script CertoraProver/certoraBuild.py to main_with_args()
Creating dir /home/manoel/projeto/GasOptimizationCatalog/certora/mappings-vs-arrays/conf/.certora_internal/25_09_21_11_29_32_207/.certora_config
In /home/manoel/projeto/GasOptimizationCatalog/certora/mappings-vs-arrays/specs/mappingarray.spec, found the imports: []
copying spec file /home/manoel/projeto/GasOptimizationCatalog/certora/mappings-vs-arrays/specs/mappingarray.spec to /home/manoel/projeto/GasOptimizationCatalog/certora/mappings-vs-arrays/conf/.certora_internal/25_09_21_11_29_32_207/.certora_sources/.0_mappingarray.spec.spec
writing /home/manoel/projeto/GasOptimizationCatalog/certora/mappings-vs-arrays/conf/.certora_internal/25_09_21_11_29_32_207/.certora_verify.json
Failed to run Certora Prover locally. Please check the errors below for problems in the specifications (.spec files) or the prover_args defined in the .conf file.
[main] ERROR ALWAYS - Found errors in ../specs/mappingarray.spec:
[main] ERROR ALWAYS - Error in spec file (mappingarray.spec:81:47): Syntax error: unexpected token near `uint256`
build failed
